NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs

```diff
@@ -58,10 +58,10 @@ public IEnumerator SpawnAsync(Entity entity, Optional<GameObject> parent, Entity
         IEnumerator CreateGameObject(TechType techType, string classId, TaskResult<GameObject> result)
         {
             GameObject prefab = null;
-            if (PrefabDatabase.TryGetPrefabFilename(classId, out string filename))
-            {
-                prefab = Resources.Load<GameObject>(filename);
-            }
+
+            IPrefabRequest prefabCoroutine = PrefabDatabase.GetPrefabAsync(classId);
+            yield return prefabCoroutine;
+            prefabCoroutine.TryGetPrefab(out prefab);
```

Comment:
Would move variable declaration inside the TryGetPrefab()

Reply:
Done